packages/vlt/vlt.0.2.3/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A variant of Bolt logging tool"
+description: """
+Vlt is yet another variant of Bolt logging tool derived from Volt.
+Unlike Volt, Vlt relies on ppx instead of camlp4 to preprocess source code."""
+maintainer: ["codinuum@me.com"]
+authors: ["Xavier Clerc" "Codinuum"]
+license: "LGPL-3.0-only"
```

Comment:
the repo has both gpl and lgpl license

should both be mentionned in the package or is the GPL code not packaged somehow?

Reply:
The [home page](https://bolt.x9c.fr/) of the original Bolt states that it is released under the LGPLv3, while the source code distribution contains both COPYING and COPYING.LESSER.
As far as we know, the original distribution and also Vlt do not contain GPL code.